code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -459,17 +462,61 @@
 	return ..()
 
 /mob/living/simple_animal/bot/screwdriver_act(mob/living/user, obj/item/I)
+	. = TRUE //Must be true or we attempt to stab the bot | Sets standard return value for this code block
+
 	if(user.a_intent == INTENT_HARM)
 		return ..()
 	if(locked)
-		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
-		return TRUE // Must be true or we attempt to stab the bot
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return
 
-	open = !open
 	I.play_tool_sound(src)
+	open = !open
 	to_chat(user, "<span class='notice'>The maintenance panel is now [open ? "opened" : "closed"].</span>")
+	return
+
+/mob/living/simple_animal/bot/wrench_act(mob/living/user, obj/item/I)
+	. = TRUE //Must be true or we attempt to stab the bot | Sets standard return value for this code block
+
+	if(user.a_intent != INTENT_HELP)
+		return ..()
+	if(locked) //ID-Lock
```

Comment:
```suggestion
	if(locked)
```
Autodoc already shows this!

Reply:
Okay.. I started a review by mistake and closing it deleted my comment.

I don't have the extension, wich autodoc shows it ? "Auto dmdoc" ?

Reply:
OH. You can just mouse over variables and it tells you.